src/components/step-sequence/step-sequence.stories.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { select } from '@storybook/addon-knobs';
+import { StepSequence, StepSequenceItem } from './step-sequence';
+import OptionsHelper from '../../utils/helpers/options-helper';
+
+storiesOf('Step Sequence', module).add('default', () => {
+  const orientation = select('orientation', OptionsHelper.orientation, OptionsHelper.orientation[0]);
```

Comment:
Can we take the default value from the component? I.e. `StepSequence.defaultProps.orientation`